invenio_rdm_records/views.py

```diff
@@ -102,3 +104,16 @@ def create_iiif_bp(app):
     """Create IIIF blueprint."""
     ext = app.extensions["invenio-rdm-records"]
     return ext.iiif_resource.as_blueprint()
+
+
+file_transfer_type_constants = {
```

Comment:
do we need this importable?
can this be moved to the `file_transfer_type` function?

Reply:
Now it should look better, less cluttered :)